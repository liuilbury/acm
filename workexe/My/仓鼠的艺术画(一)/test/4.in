55 7 62
9 6
30 6
53 6
53 7
41 1
13 4
7 7
3 7
40 6
26 5
32 3
11 2
36 3
13 4
48 6
31 1
54 3
40 6
25 4
11 4
42 1
16 7
8 3
36 2
6 2
53 4
22 4
41 4
48 1
22 4
19 3
18 7
11 1
33 7
13 1
55 4
48 1
25 6
48 1
8 5
24 5
1 4
40 7
42 7
22 3
15 1
13 7
48 5
20 6
54 3
27 2
49 1
30 2
31 7
47 4
38 4
35 1
28 2
42 2
26 3
33 2
24 5

