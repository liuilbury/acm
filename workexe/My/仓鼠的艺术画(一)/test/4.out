* * 42 42 42 * *
* * * 42 * * 8
* * * * * 8 8
* * * * * * 8
* 25 * * * * *
25 25 25 * * * 7
* 25 23 * 40 7 7
* 23 23 63 40 41 7
* * 23 * 41 1 1
33 12 * 20 * 1 *
45 45 32 20 20 * *
68 12 * 40 * * 47
35 35 20 20 20 47 47
81 * * 20 * * 47
46 46 * * * * 22
46 * * * * 22 22
* * * * * * 54
* * 31 * * 32 32
* 31 31 31 * 49 32
* * 31 * 49 49 49
* * 45 57 * 49 *
* 45 102 102 57 * *
* * 45 57 103 * *
* * * 122 103 141 *
* * 79 19 170 38 38
* 111 60 89 10 48 *
51 109 111 * 10 * *
58 109 58 * * * *
* 111 * * * 2 *
69 53 53 * 2 2 56
16 69 11 * * 56 54
16 72 11 11 * * 88
61 61 72 * * 34 34
57 61 * * * * 34
57 81 13 * * * *
81 37 37 13 * * *
* 24 13 56 * * *
* * 56 56 56 * *
* * * 56 * 27 43
5 * * 28 27 70 70
26 64 28 28 28 27 87
85 80 59 28 * 44 44
21 59 * * * * 44
* * * * * * *
* * * * * * *
* * * 55 * * *
105 * 55 55 103 15 *
157 105 * 103 63 63 15
157 52 * * 48 15 *
52 * * * * * *
* * * * * * *
* * * 26 * 3 4
* * 93 26 29 7 7
* 67 67 129 * 3 4
* * 103 36 36 * *
